empress/support_files/js/colorer.js

```diff
@@ -553,6 +565,8 @@ define(["chroma", "underscore", "util"], function (chroma, _, util) {
         "#808000",
         "#008080",
     ];
+    Colorer.__PR_OR_GR = "GnOrPr";
+    Colorer.__prOrGr = ["#008000", "#fed8b1", "#800080"];
```

Comment:
Would it be possible to change these to reflect the new name? (This'd also involve updating the other uses of these variables throughout this module.)

This is not very high priority, since at this point it's just to avoid confusing future people looking at this code who wonder why the names are switched.

```suggestion
    Colorer.__GN_OR_PR = "GnOrPr";
    Colorer.__gnOrPr = ["#008000", "#fed8b1", "#800080"];
```

Reply:
good catch! I'll change them.